src/quo2/components/tags/context_tags.cljs

```diff
@@ -47,15 +47,17 @@
                  :size :paragraph-2}
       (trim-public-key public-key)]]))
 
+(defn context-tag [params photo name]
```

Comment:
I had to use a context tag in this p.r as that's what matches the designs but it's not dependent on the user, rather the community.

Reply:
If it's the same UI we can abstract the look